File: src/main/python/LineCount.py
import sys
from pyspark import SparkContext

if __name__ == "__main__":
  if len(sys.argv) < 2:
    print >> sys.stderr, "Usage: LineCount <directory|file>"
    exit(-1)
  sc = SparkContext()
  count = sc.textFile(sys.argv[1]).count()
  print "Line Count: ", count

File: src/main/scala/my/spark/examples/LineCount.scala
package my.spark.examples

import org.apache.spark.SparkContext

object LineCount {
  def main(args: Array[String]) {

    if (args.length < 1) {
      System.err.println("Usage: LineCount <directory|file>")
      System.exit(1)
    }
    val sc = new SparkContext()
    val count = sc.textFile(args(0)).count()
    println( "Line Count: " + count)
    sc.stop()
  }
}
